pkgs/top-level/all-packages.nix

```diff
@@ -2923,6 +2923,7 @@ in
 
   dleyna-server = callPackage ../development/libraries/dleyna-server { };
 
+  dmdBootstrap = callPackage ../development/compilers/dmd/bootstrap.nix { };
```

Comment:
I am not sure if it is good to provide the bootstrap package this way.
It should not be available to the outside because it is not a "nice" package but a hackish one and is only needed for building the real package which provides the same functionality.
I can only imagine a testing scenario where having the bootstrap package available that way but this is highly speculative as I don't have a concrete example.
Even if such an example could be found it's for sure quite rare and can be added by a developer temporarily without polluting the namespace.

Reply:
removed.